src/main/java/com/box/sdk/BoxMetadataFilterRequest.java

```diff
@@ -0,0 +1,29 @@
+package com.box.sdk;
+import com.eclipsesource.json.JsonArray;
+import com.eclipsesource.json.JsonObject;
+/**
+ * <p>BoxMetadataFilterRequest class was designed to wrap the BoxMetadataFilter close.
+ * Best practice is to use the two in conjuction.</p>
+ *
+ */
+public class BoxMetadataFilterRequest {
```

Comment:
Does this class need to be exposed to the user? i.e., are there situations where they would need to use this to create the JSON?

If not, we could consolidate this code into the BoxSearchParameters class.